### ISSUE TITLE
Missing 0.2.0 in opam-repository

### ISSUE THREAD

**Issue by kit-ty-kate** [ocaml/stdlib-shims#18]
The version 0.2.0 is not currently in opam-repository. However it is going to be required for the OCaml 4.12 support. Was there something stopping it to be fully released?

**Comment by kit-ty-kate** [comment]
*oops clicked tapped enter too quickly, I just edited the main description to give a bit more details*

**Comment by dra27** [comment]
Yes, the original one got pulled because of a problem - we’ll get a proper replacement cut soon!

**Comment by chetmurthy** [comment]
Hi, any news on when this might be released?  If there's a pointer to the problem, maybe some volunteer (me) could help?

**Comment by kit-ty-kate** [comment]
The blocker is dune which needs a release (2.8.0) to be compatible with OCaml 4.12.
